Update Project System.md

pmoncada opened the pull request:
<!-- CURSOR_SUMMARY -->
> [!NOTE]
> Overhauls Project System to v8 with quarterly proposal/voting flow, updated roles/responsibilities, revised rewards calculation/payouts, and expanded FAQs.
> 
> - **Docs/Governance**:
>   - Update `Project System.md` to **v8.0** with revised terminology (e.g., Executive Branch Leads) and clearer mission phrasing.
> - **Proposal & Voting Process**:
>   - Introduce quarterly deadlines: submission by second Thursday; Senate review; townhall presentation; Member House vote with funding selection rules and 1/5 budget cap per proposal; 3/4 quarterly budget cap enforcement.
> - **Project Start & Operations**:
>   - Clarify payment streaming options for upfront payments.
>   - Shift to monthly progress updates; require weekly townhall attendance with 5% penalty per miss; rename contributor update channel.
> - **Completion**:
>   - Add Executive Branch Leads review/grade of final reports; successful leads may join Senate for 6 months.
> - **Quarterly Rewards**:
>   - ETH reward now 5% of NMA minus total project budgets (excl. Executive Branch Budget); clarify vMOONEY geometric series formula.
>   - Run rewards script per project; remove contributor-level reward formula; send ETH rewards to project multisig; clarify vMOONEY vesting.
> - **FAQs**:
>   - Add/clarify: contributor payments must be streamed (recommend llamapay); budget limits; multi-quarter milestones; DAO-created multisigs; amendment authority moved to Executive Branch Leads.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 160582b0ccbeb80ea22a595b94f8c455d59e4272. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>
<!-- /CURSOR_SUMMARY -->

pmoncada committed:
Update Project System.md

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -38,7 +38,11 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go f ...
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then  ...
+
+**Step 3: Approval.** All valid proposals that have been approved by the Senate will then be voted on via ranked-choice vote by the Member House after they are presented during the townhall. The top 25% of proposals that quarter will be funded.
```
RCV will reallocate votes until ONE achieves 50% -- it's good for 1 winner take all vote like President, but not really apt for this, I think we'd need to use weighted vote so we can select multiple projects and have multiple winners.

pmoncada replied:
Weighted sounds fine to me -- what I was thinking was that you could "eliminate" options via the same RCV algorithm until you get the top 25% (just pause at 25% instead of continuing until there's just one), but no need to overcomplicate it.

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -50,7 +54,7 @@ The Project’s Lead becomes a member of the **Senate** as outlined in our DAO C
 Throughout the lifetime of the Project the **Lead** must do the following:  
 
-**1. Provide Weekly Progress Updates to the Senate.** Every week the team must provide a written report and present it at the townhall meeting. This is intended to be a helpful meeting to give context to the DAO on progress and also have the DAO find ways to support the project. However, if the project is not meeting the expectations of their project or no progress has been made, then the Astrona ...
+**1. Provide a Monthly Progress Updates to the Senate.** Every month the team must provide a written report and present it at the townhall meeting. This is intended to be a helpful meeting to give context to the DAO on progress and also have the DAO find ways to support the project. However, if the project is not meeting the expectations of their project or no progress has been made, then the Ast ...
```
typo > Monthly vs A Monthly Progress Updates.

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -80,7 +84,7 @@ Each quarter, the projects that were completed will be eligible for Project Rewa
 The total amount of rewards each quarter is calculated as follows:
 
-**1. ETH Reward:** 5% of the liquid non-MOONEY assets (NMA) (e.g. ETH, DAI, and other stables) paid in the form of ETH. Assets with fluctuating prices will be priced at midnight of the first day of the quarter.
+**1. ETH Reward:** 5% of the liquid non-MOONEY assets (NMA) (e.g. ETH, DAI, and other stables), minus the sum total of all the project budgets for that quarter. Paid in the form of ETH. Assets with fluctuating prices will be priced at midnight of the first day of the quarter, 
```
all non MoonDAO operations projects? If we include EB quarterly subtraction there will be zero quarterly ETH.

pmoncada replied:
Yeah not including EB

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -137,17 +136,33 @@ If Dispute Resolution through MoonDAO’s arbitration procedure is unsuccessful,
 **What happens if the project misses its deadline?**
 
-If you are going over the originally intended timeline please notify the Senate in your monthly update. If the deadline extends into a new quarter the Astronauts will have to approve the project’s continuation.  
+If you are going over the originally intended timeline please notify the Senate in your monthly update. If the deadline extends into a new quarter the Astronauts will have to approve the project’s continuation. This is not unexpected, the project will still be eligible for retroactive rewards in the next quarter.
```
Take the opportunity to change Astronauts > Executive Leads

pmoncada committed:
Update Project System.md

pmoncada committed:
Update Project System.md

pmoncada committed:
Update Project System.md

Change to top 50%, or top 4 proposals

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -38,7 +38,11 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go f ...
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then  ...
+
+**Step 3: Approval.** All valid proposals that have been approved by the Senate will then be voted on via weighted vote by the Member House after they are presented during the townhall. The top 50% of proposals that quarter will be funded, or if less than 8 proposals are considered, the top four proposals will be funded.
```
1. Do the constraints on how the duration of a voting window belong here? or is it a matter of norms/EB discretion? 

2. Given the cap of 5% treasury, is there a cap on the total spend allowed per quarter? Thinking of the extreme cases, what if the top 50% of projects sum total funding exceeds the total spending budget that can be paid out by the treasury? I imagine that it would end up looking like a scaled but weighted payout to the selected projects such that the total stays within the DAO's means. I see some weird pinches coming up with the unbounded "top 50%

pmoncada replied:
1.) Added a 5 day voting period to match what we have in the Constitution. It's already there, but no problem mirroring it here as well to be explicit.
2.) Yes, the total maximum budget for that quarter would what's in the retroactive rewards. If the passed projects cumulative budget is greater than that amount... we'd need to either:

a.) Scale budgets down (like you mention, which might be a little complicated and affect all projects and reduce their budget).
b.) Accept less projects (maybe introduce a budget limit alongside the 50%)

I think (b.) makes sense to me, and then we could also add a funding maximum (i.e. no project can have more than X% of the quarterly budget). Say... max of 20% of the budget for that quarter. So if the retros are $25k that quarter, we'd have a $5k max budget / project. Then they'd get the rest in retroactives.

20% cap means we will have at least have 5 projects per quarter.

pmoncada replied:
Updated for the total up-front cumulative budget to be 3/4, so that no matter what there's still some left over for retroactive rewards.

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -50,7 +54,7 @@ The Project’s Lead becomes a member of the **Senate** as outlined in our DAO C
 Throughout the lifetime of the Project the **Lead** must do the following:  
 
-**1. Provide Weekly Progress Updates to the Senate.** Every week the team must provide a written report and present it at the townhall meeting. This is intended to be a helpful meeting to give context to the DAO on progress and also have the DAO find ways to support the project. However, if the project is not meeting the expectations of their project or no progress has been made, then the Astrona ...
+**1. Provide monthly progress updates to the Senate.** Every month the team must provide a written report and present it at the townhall meeting. This is intended to be a helpful meeting to give context to the DAO on progress and also have the DAO find ways to support the project. However, if the project is not meeting the expectations of their project or no progress has been made, then the Execu ...
```
Perhaps some more explicit language around what is required for a monthly report? like progress toward OKRs, realized risks, total spend so far etc... this is another thing that I'm sure is covered by norms - floating the idea of codifying it. Might be a "to add somwday" thing but okay to omit for now

pmoncada replied:
Yeah - good thought. I think we can leave this open ended for now. I'm thinking something like a presentation with a few required things (how OKRs are going so far etc.)

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -74,13 +78,13 @@ The Project Lead must make sure the following are complete:  
 
 **Total Quarterly Reward Calculation**
 
 The total amount of rewards each quarter is calculated as follows:
```
If its no trouble to add, defining the quarter boundaries explicitly would make the project system more grounded. so many things depend on the "end of the quarter" without clarifying where MoonDAO draws the line. for example, if the dates slide into the "last whole week of the calendar quarter" instead of ending on a Monday or whatever, that can make a big difference

pmoncada replied:
These are drawn explicitly in the Constitution

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -74,13 +78,13 @@ The Project Lead must make sure the following are complete:  
 The DAO incentivizes people to start new projects with its quarterly rewards system.  
 
-Each quarter, the projects that were completed will be eligible for Project Rewards if and only if they have submitted their Final Report to the DAO and it has been approved by the Astronauts. Final reports must be submitted by the end of the quarter.
+Each quarter, the projects that were completed will be eligible for Project Rewards if and only if they have submitted their Final Report to the DAO and it has been approved by the Executive Branch Leads. Final reports must be submitted by the end of the quarter.
```
A thought for later: these submissions and/or approvals ought to be on chain. The initial vote opens the "contract" and these approvals close it

pmoncada replied:
They are to some extent already, this status is writte onchain in the projects table.

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -74,13 +78,13 @@ The Project Lead must make sure the following are complete:  
 The total amount of rewards each quarter is calculated as follows:
 
-**1. ETH Reward:** 5% of the liquid non-MOONEY assets (NMA) (e.g. ETH, DAI, and other stables) paid in the form of ETH. Assets with fluctuating prices will be priced at midnight of the first day of the quarter.
+**1. ETH Reward:** 5% of the liquid non-MOONEY assets (NMA) (e.g. ETH, DAI, and other stables), minus the sum total of all the project budgets for that quarter (not including the Executive Branch Budget). Paid in the form of ETH. Assets with fluctuating prices will be priced at midnight of the first day of the quarter, 
```
Treasure.Corp dashboard to track this? 👀

pmoncada replied:
We track this right now in the /projects tab - we'll surface this better.

philiplinden commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -38,7 +38,11 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go f ...
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then  ...
+
+**Step 3: Approval.** All valid proposals that have been approved by the Senate will then be voted on via weighted vote by the Member House after they are presented during the townhall. The top 50% of proposals that quarter will be funded, or if less than 8 proposals are considered, the top four proposals will be funded.
```
I think its time to codify the norms around project participants abstaining from their own projects' votes.

pmoncada replied:
Agreed. Will add.

pmoncada committed:
Specify voting period for proposal approval

Added a five day voting period to the approval process for proposals.

pmoncada committed:
Update project proposal submission and approval steps

Added budget constraints for project proposals and clarified funding process.

pmoncada committed:
Update approval process for project proposals

Clarified the approval process for project proposals, including voting criteria and contributor voting restrictions.

pmoncada commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -38,7 +38,11 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go f ...
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then  ...
+
+**Step 3: Approval.** All valid proposals that have been approved by the Senate will then be voted on via weighted vote by the Member House after they are presented during the townhall, with a five day voting period. The top 50% of proposals that quarter will be funded, or if less than 8 proposals are considered, the top four proposals will be funded.
```
Fixed the wording for this... thanks Cursor :)

pmoncada committed:
Increase maximum proposal budget to 20%

Updated the maximum proposal budget from 5% to 20% of the total retroactive reward for the quarter.

pmoncada commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -137,25 +136,41 @@ If Dispute Resolution through MoonDAO’s arbitration procedure is unsuccessful,
+
+**How much can I ask for?**
+
+Proposal budgets should not exceed the total retroactive reward for that quarter. In other words, it should not be more than 5% of the treasury. If you are looking for more budget than that, consider using our launchpad to help raise funds from the community.
```
Updated this to be consistent with another edit. Thanks again cursor.

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -107,17 +111,12 @@ We multiply by 0.9 because 10% of the total rewards always goes to the Contribut
+The ETH will be given as a lump-sum reward within a month of the end of the quarter, and sent to the project's multisig.
  
-The MOONEY will be locked for four years and given in the form of delegated vMOONEY that they can unlock after it has vested.
+The MOONEY will be locked for four years and given in the form of delegated vMOONEY that they can unlock after it has vested, and sent to individual contributors directly.
```
We don't actually use delegated vMOONEY anymore, change delegated to locked?

pmoncada replied:
Good catch!

ryand2d commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -38,19 +38,23 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
 This is meant to help YOU get feedback from people on the idea, offer alternative solutions, and allow for informal coordination around ideas. Anyone can post about ideas regardless of their background in the DAO, even if they aren’t yet part of our community.
 
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go f ...
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then  ...
```
24 hrs before 3rd town hall -- do we need more time? If we want to workshop proposals or have a private senate review before presentation... Right now, I've got Jan 15th penciled in for the Town Hall presentations which is the third thursday of the month but not third town hall, since Jan 1 is a holiday.

pmoncada replied:
Yes I still think this is a good idea. From our conversation this morning, we'll have the following schedule:

- 1 week before the 3rd townhall is the deadline for proposals. 
- Senate convenes privately to review proposals the next day, gives feedback.
- 48 hours before 3rd townhall is the deadline for a final draft.
- Senate votes yes/no to include for consideration that quarter 24hrs in advance.
- Projects Present the proposals in the townhall, token holders vote.

I'll add this soon

rdbarnum commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -80,7 +84,7 @@ Each quarter, the projects that were completed will be eligible for Project Rewa
+**1. ETH Reward:** 5% of the liquid non-MOONEY assets (NMA) (e.g. ETH, DAI, and other stables), minus the sum total of all the project budgets for that quarter. Paid in the form of ETH. Assets with fluctuating prices will be priced at midnight of the first day of the quarter, 
 
 **2. vMOONEY Reward:** A geometric series of MOONEY will be released as follows:
 1. Q4 2022: 15,000,000 MOONEY. 
```
Outside the scope of the current changes, but technically we are decreasing by 0.05 every quarter in MOONEY rewards. 

R[Q_n] = 0.95*R[Q_n-1], where R[Q_n] means the MOONEY reward for any given quarter "n", might be more clear for point five (old line 90, new line 94)

pmoncada replied:
I'll add this for extra clarity - it's probably good to be totally explicit about the formula.

rdbarnum commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -137,17 +136,33 @@ If Dispute Resolution through MoonDAO’s arbitration procedure is unsuccessful,
+
+**How much can I ask for?**
+
+Proposal budgets should not exceed the total retroactive reward for that quarter. In other words, it should not be more than 5% of the treasury. If you are looking for more budget than that, consider using our launchpad to help raise funds from the community.
```
We should probably add some math or language around what to do when  Σ (top 25% project budgets) > 5% of the treasury ->

Two options:
1) reduce all funding proportionally (less than ideal probably for each project, but equal)
2) drop lowest project(s) until total budget of projects is < 5%  (this might be better, as no funded projects are under funded, but means we will be limited to less than top 25% in some circumstances)

pmoncada replied:
Ah I think you might be looking at an outdated version of this - check out the latest commits, this was updated to account for this! See the discussion here: https://github.com/Official-MoonDao/documentation/pull/112#discussion_r2557269605

pmoncada replied:
The chosen solution was #2 (reasoning in the comment), but I'd be interested to hear your thoughts about 1 v.s. 2!

rdbarnum replied:
I think #2 is the most logical approach. While it does mean funding slightly fewer projects, I think it ensures that no project goes under funded per their original funding requests.

pmoncada committed:
Update MOONEY reward calculation formula

Clarified the formula for MOONEY rewards in Project System.

pmoncada committed:
Clarify proposal budget limits in Project System.md

pmoncada committed:
Update MOONEY reward distribution details

Clarified the locking and unlocking process for MOONEY rewards.

pmoncada committed:
Refine project proposal process and remove redundancy

Removed redundant text and clarified steps for project proposals and approvals.

pmoncada committed:
Refine Project System documentation for clarity

Clarified roles and responsibilities of Project Lead and Contributors, updated proposal review process, and improved language for better understanding.

pmoncada committed:
Update project meeting requirements and penalties

jaderiverstokes commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -35,28 +34,30 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
+**Step 3: Senate Review and Approval.** The Senate will convene and review incoming proposals and provide feedback the next day. Proposals may be edited up to **48 hours** before the third Thursday of the Quarter. After the editing deadline is over, the Senate will approve or reject the proposals in accordance with the Constitution. If a proposal passes it will move on to the next step.
 
-## Start
+**Step 4: Townhall Presentation and Member House Vote.** Project contributors MUST be present during the townhall to pitch their project proposal to the community. The member house will then vote on proposals in accordance with the Constitution. The top 50% of proposals that quarter will be funded ranked in order of voting power allocated to them. If less than six proposals are considered, the to ...
```
I don't see how the vote actually works here?

I think it's a new process so it should be described somewhere.

rdbarnum replied:
Does Step 3 mean there is a mandatory (separate) Senate meeting? Does this need to be live/recorded with deliberations? Will we still vote using temp check or does there need to be a new (auditable) voting process?

pmoncada replied:
Yes there will be a separate Senate meeting for giving feedback to all incoming proposals. This one will not be recorded, attended only by the Senate and EB leads, and no vote will happen during this meeting. It is intended to be a long-form discussion and feedback session, with the outcome being comments and feedback on each of the proposals by senate members.

This format also allows for Senators and EB leads to discuss proposals freely with candid discussion about the pros/cons without inadvertently offending the authors of the proposals, but then structuring feedback on the proposal itself in a constructive and helpful way.

Again, no vote happens here, it's just intended to be a discussion and written feedback session for proposals.

pmoncada replied:
Also @ jade: No vote happens here. Regular temp check senate vote happens after the proposal is "finalized" 48 hours before the townhall. Then regular senate voting happens like we currently do, but before the townhall, so that only the proposals that have passed the senate are discussed at the townhall

jaderiverstokes replied:
Sorry my comment was ambiguous. I'm referring to the member house vote. I think we should have an explanation of how that vote works. It's a new voting process, so worth spelling out explicitly. Something like:

"Members vote by allocating voting power on a percentage basis across approved projects."

pmoncada replied:
Added to be more explicit!

pmoncada committed:
Update Townhall Presentation and Voting Process

Clarified funding criteria and voting process for project proposals.

pmoncada committed:
Clarify voting process in project proposal steps

pmoncada committed:
Revise Step 4 for clarity on voting process

Clarified the voting process for project proposals during the townhall, including how voting power is allocated and adjusted based on the number of proposals.

jaderiverstokes commented on MoonDAO/docs/Projects/Project System.md:
```diff
@@ -137,25 +135,41 @@ If Dispute Resolution through MoonDAO’s arbitration procedure is unsuccessful,
+
+**How much can I ask for?**
+
+Proposal budgets must be less than or equal to 1/5 of the total quarterly budget. If you are looking for more budget than that, consider using our launchpad to help raise funds from the community.
```
Nit: can we put this above where the rest of the budget is discussed.

pmoncada replied:
Added in step 2. It's also in the project proposal template, but added here as well for more clarity

jaderiverstokes commented:
Love the proposal. We should definitely be switching things up and seeing what works best. Whether this setup is successful or not is a matter of trying it out and getting some data. So I think we prioritize simplicity so that we feel liberated to run experiments.

I think the opening paragraph is pretty clear on the goal for projects:
"MoonDAO Projects are goal-oriented teams working on mission-aligned objectives. This document provides the framework to fund the team, support their progress, align their interests with the DAO, and provide retroactive incentives for projects that get us one step closer to a permanent human presence on the Moon."

However I think there's a lot of room for interpretation as to which projects support the mission, and how we determine if they are successful. Maybe we could make it more concrete with metrics. For example, we make a rubric and score each project across categories on a scale from 1-10. Categories could be: ROI, initiative, innovation, reach, etc. Without some sort of definite target, I fear that each project stays isolated. Maybe individually they look like they are mission focused, but after a long period of time we find we are no closer to our goal.

Overall tho, I think the update is good to go as is, just some thoughts on other additions.

pmoncada commented:
> -

Good thoughts Jade - maybe we could incorporate the rubric in the Senate review. We used to have a rubric, but ultimately people didn't really spend the time to fill it out. Although now with the separate Senate meeting it might be worth bringing that back in, that rubric can be more of an "implementation detail" though that doesn't need to live in this doc.

pmoncada committed:
Update Project System.md with proposal guidelines

Added guidelines for proposal limits and budget constraints in the Project System documentation.
